### PR TITLE
[dev-v2.8] rancher-logging 103.1.0+up4.4.0 update

### PR DESCRIPTION
**Pull request by HarrisonWAffel**
## Problem

Various components of Rancher are using Windows images which contain Foreign Layer references. This makes mirroring these images more difficult and is a common source of user confusion when configuring windows nodes in air-gapped environments. Other PRs to rancher/rancher and rancher-monitoring have already resulted in many of these images being fixed, however logging has not yet been addressed. 

## Solution
+ Bump rancher/fluent-bit to `v1.8.10`
    + `v1.8.10` is simply a rebuilt version of `1.8.9` which no longer contains foreign layer references. There is no functional difference between the two.
+ Remove `rancher-logging 103.1.0-rc1+up4.4.0`
+ Adds `rancher-logging 103.1.0-rc2+up4.4.0`

## Testing
I've confirmed the chart can be installed and works as expected. I've confirmed that `v1.8.10` of rancher/fluent-bit does not include any foreign layer references.

### CODE REVIEW

**Review by joshmeranda — CHANGES_REQUESTED** [review]
Just fix the merge conflicts, otherwise LGTM

**Review by joshmeranda — APPROVED** [review]

File: regsync.yaml
Comment (lucasmlp) [inline regsync.yaml:1]: This file should not be changed by contributors. We have a GitHub action that automatically detects new docker images and updates `regsync.yaml`

**Review by lucasmlp — CHANGES_REQUESTED** [review]

File: regsync.yaml
Comment (HarrisonWAffel) [reply regsync.yaml:1]: I've removed the changes to `regsync.yaml`.

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Pull request merged**